server/server.js

```diff
@@ -177,14 +176,27 @@ function sendCorsHeaders(req : ExpressRequest, res : ExpressResponse) {
     if (corsRequestMethod) {
         res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_METHODS, corsRequestMethod);
     }
+
+    res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_CREDENTIALS, 'true');
 }
 
 export function expressEndpoint({ uri = '/', logger = defaultLogger, enableCors = false } : ExpressEndpointOptions = {}) : mixed {
 
     // $FlowFixMe
     const app = require('express')();
+    const bodyParser = require('body-parser');
+    
+    app.use(bodyParser.text());
```

Comment:
Should we also add `bodyParser.json()` for the fallback xhr use case? When testing locally, I had to add it to see the data flow through on the server.

Reply:
Interesting.  This wasn't there before for the request being `application/json`.  Will check.

Reply:
Good deal. I was testing by adding the following console.logs here in server.js:
```js
// assuming both the sendBeacon() and xhr post return true here
if (method.toLowerCase() === 'post') {
        const { events, tracking, meta } = body;
        console.log('events', events);
        console.log('tracking', tracking);
        console.log('meta', meta);
        // console.log('req.body', req.body);
        log(req, logger, { events, tracking, meta });
    } else {
        const { event, level = LOG_LEVEL.INFO, ...payload } = query;

        log(req, logger, {
            events: [
                {
                    level,
                    event,
                    payload
                }
            ]
        });
    }
```

Reply:
@bluepnume we went ahead and added the json body parser as well (`app.use(bodyParser.json())`). Is that cool with you? My guess is the parent express app consuming this is doing `bodyParser.json()` as well which is why this isn't a bug with the old xhr implementation. We figured it was safest to add it here as well.